beacon-chain/state/stateutil/attestations.go

```diff
@@ -88,6 +88,64 @@ func marshalAttestationData(data *ethpb.AttestationData) []byte {
 	return enc
 }
 
+func attestationRoot(att *ethpb.Attestation) ([32]byte, error) {
+	fieldRoots := make([][32]byte, 3)
+
+	// Bitfield.
+	aggregationRoot, err := bitlistRoot(att.AggregationBits, 2048)
```

Comment:
Instead of hard coding 2048, better to use `params.BeaconConfig.MaxValidatorsPerCommittee`

It changes to 1024 and it'll take next person a while to discover this is the issue :)

Reply:
ah nice, catch. changing it now